hello_world.cpp

```diff
@@ -0,0 +1,7 @@
+// Note: this file is just to mark this project as C++ related.
+#include <iostream>
+
+int main()
+{
+    std::cout << "Hello, world!\n";
```

Comment:
It's not necessary, actually std::endl is in present often overused.
The main difference is that std::endl make a flush/sync of stream. This will be done automatically when program ends.